woodwork/tests/accessor/test_statistics.py

```diff
@@ -245,6 +245,25 @@ def __call__(self, update, progress, total, unit, time_elapsed):
     assert mock_callback.total_elapsed_time > 0
 
 
+def test_get_valid_mi_columns(df_mi):
+    df_mi.ww.init()
+    valid_columns = df_mi.ww.get_valid_mi_columns()
+    mi = df_mi.ww.mutual_information()
+    valid_mi_columns = (mi.column_1.append(mi.column_2)).unique()
+
+    assert (valid_columns == valid_mi_columns).all()
+
+
+def test_get_valid_mi_columns_with_index(sample_df):
+    sample_df.ww.init(index='id')
+    mi = sample_df.ww.get_valid_mi_columns()
```

Comment:
If this line is specifically supposed to produce an `mi` result where the index is not included, we should probably do this:
```suggestion
    mi = sample_df.ww.get_valid_mi_columns(include_index=False)
```
Even if that is already the default value, it makes the intentions of the test case clear.